tests/ops/test_qubit_ops.py

```diff
@@ -989,9 +989,9 @@ def circuit(theta):
             return qml.expval(qml.PauliZ(0))
 
         res = circuit(angle)
-        gradient = np.squeeze(circuit.jacobian(angle))
+        gradient = np.squeeze(qml.grad(circuit)(angle))
```

Comment:
This is because of the change in the shape of the returned object?

Reply:
No, this is because, in tape mode, the QNode no longer has a `jacobian` method. It makes sense to instead use the user facing API in an integration test, that way it is immune to low level implementation details.

Reply:
I am also a bit confused here, shouldn't `qml.jacobian` be rather used to replace `circuit.jacobian`?

Reply:
Oh, yep! I changed this to `grad` simply because the QNode has one parameter and one input lol. It also matches the docstring of the test better.